docs/reference/release-notes/850.md

```diff
@@ -0,0 +1,188 @@
+---
+id: 850
+title: "8.5 Release notes"
+description: "Release notes for 8.5, including alphas"
+---
+
+## Enhancements
+
+### Role Support for Web Modeler with OIDC in SM
```

Comment:
Maybe this is another backlog item for me as I'm not sure if we do it anywhere else, but I'd probably avoid abbreviations of SM 👍

Reply:
Oops, I thought I caught all of them - let's definitely adjust this.

Reply:
Fixed